traitsui/qt4/list_editor.py

```diff
@@ -382,6 +382,21 @@ def _mutable_default(self):
         """
         return self.factory.mutable
 
+    def _scrollable_default(self):
```

Comment:
You don't have a `scrollable` trait declared on the list editor (as opposed to the list editor factory).  So this _probably_ isn't working the way that you expect.

Reply:
The `scrollable` was present in the parent class `UIEditor` -- but I realize now that it is set to `False` by default. Setting default to `True` here.